.stylelintrc

```diff
@@ -0,0 +1,68 @@
+{
+  "extends": "stylelint-config-recommended",
+  "plugins": [
+    "stylelint-declaration-block-no-ignored-properties",
+    "stylelint-order",
+    "stylelint-scss"
+  ],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "block-no-empty": true,
+    "color-hex-case": "lower",
+    "color-hex-length": "long",
+    "color-named": "never",
+    "comment-no-empty": true,
+    "declaration-block-no-shorthand-property-overrides": true,
+    "declaration-block-no-duplicate-properties": true,
+    "declaration-block-no-redundant-longhand-properties": true,
+    "declaration-block-trailing-semicolon": "always",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-unless-keyword",
+    "font-family-no-duplicate-names": true,
+    "font-weight-notation": "numeric",
+    "function-calc-no-invalid": true,
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always",
+    "function-name-case": [
+      "lower",
+      ignoreFunctions: ["containerWidth", "lineHeight", "zIndex", "fontSizesByBreakpoint", "withNElements"]
+    ],
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+    "indentation": 2,
+    "length-zero-no-unit": true,
+    "max-nesting-depth": 2,
+    "no-descending-specificity": true,
+    "no-duplicate-at-import-rules": true,
+    "no-duplicate-selectors": true,
+    "no-extra-semicolons": true,
+    "no-invalid-double-slash-comments": true,
+    "no-unknown-animations": true,
+    "order/properties-alphabetical-order": true,
+    "property-no-unknown": true,
+    "scss/at-extend-no-missing-placeholder": true,
+    "scss/at-import-no-partial-leading-underscore": true,
+    "scss/at-import-partial-extension-blacklist": ["scss"],
+    "scss/at-rule-no-unknown": true,
+    "scss/dollar-variable-colon-space-after": "always",
+    "scss/dollar-variable-colon-space-before": "never",
+    "selector-pseudo-class-no-unknown": true,
+    "selector-pseudo-element-no-unknown": true,
+    "selector-type-no-unknown": true,
+    "selector-max-combinators": 2,
+    "selector-max-compound-selectors": 2,
+    "selector-max-id": 0,
+    "selector-max-specificity": "0,2,0",
```

Comment:
`0,2,0` mi sembra un po poco, soprattutto con BEM, potremmo portare il massimo a 3?

Reply:
In BEM solitamente le dichiarazione di classi dovrebbero sempre essere: 

```
.block {}
.block__element {}
.block__element__element {}
---------------------------
.block--modifier .block__element {}
.block__element__element--modifier {}
```

Se vogliamo seguire prettamente BEM possiamo decidere di aggiungere anche questo lint in caso: https://github.com/simonsmith/stylelint-selector-bem-pattern